### DISCUSSION TITLE
[jasmine-ajax] JasmineAjaxResponse.response type should be any

### DISCUSSION THREAD

**Discussion by rooby** [DefinitelyTyped/DefinitelyTyped#56778] [Issues with a @types package]
Currently the JasmineAjaxResponse interface has 
`response?: string | undefined;`

However it should either be `any`, as it needs to match to match ArrayBuffer | Blob | Document | XMLDocument | string | null  as well as any possible JSON object. That would also make it match XMLHttpRequest's response property.

There are also no unit tests for setting JasmineAjaxResponse.response to any of ArrayBuffer, Blob, Document, XMLDocument, JSON object, which should just be passed through as is.

[a comment by an automated account is not shown]

**Comment by jgonggrijp** [comment]
I opened an issue for this.
